### ISSUE TITLE
Is it possible to scroll two separate carousels by same amount of scroll offset ?

### ISSUE THREAD

**Issue by kulik88** [nicklockwood/iCarousel#231]
I'm working on a project where I have two separate horizontal carousels (upperCarousel, lowerCarousel)  and I would like to synchronize their scroll movement. What I mean is that when I scroll one of those carousels the other should move exactly the same way. I managed to do it (partially) by assigning their scroll offset values to each other but I encountered an issue. When carousels don't wrap there is a problem when I scroll one of them to far and it starts to bounce. When that happens even after assigning upper carousel scroll offset to equal the lower carousels scroll offset it doesn't scroll. Is there a way to allow the upper carousel to move exactly the same way as the lower one even with bouncing deceleration etc. ?? I hope I explained it clearly enough.

**Comment by mbcoder17** [comment]
@kulik88 were you able to synchronize the 2 carousels?

**Comment by nicklockwood** [comment]
Yes, you can sync them perfectly. In the carouselDidScroll delegate event of one carousel set the scrollOffset of the other carousel to match the first one (the scrollOffset is now readwrite and public)

**Comment by mbcoder17** [comment]
@nicklockwood Thanks so much, that worked perfectly!

**Comment by phuhk** [comment]
@nicklockwood  hi there, so any similar solution for sync with other UIScrollView?

**Comment by nicklockwood** [comment]
@phuhk you can use the same solution, you just need to multiply the offset by some constant because iCarousel.scrollOffset uses normalized offset units and UIScrollView.contentOffset is in points.
